testng-core/src/test/java/test/configuration/ConfigurationTest.java

```diff
@@ -29,4 +31,11 @@ public void testSuite() {
     testConfiguration(SuiteTestSample.class);
     Assert.assertEquals(Arrays.asList(1, 2, 3, 4, 5), SuiteTestSample.m_order);
   }
+
+  @Test
+  public void testSuiteRunnerWithDefaultConfiguration() {
+    TestNG testNG = create(SuiteRunnerIssueTestSample.class);
+    testNG.run();
+    Assert.assertNotNull(SuiteRunnerIssueTestSample.suiteRunner);
```

Comment:
@bj-9527 - This assertion looks suspicious. Can we change this assertion such that it checks if the `run()` was successful by checking the exit code instead ?

Reply:
OK, I have change Assert to check status